### ISSUE TITLE
[QUESTION] - Change Image Dynamically

### ISSUE THREAD

**Issue by sinandomeng** [zhanziyang/vue-croppa#99]
Hi @zhanziyang, just wondering how to implement changing of initial image dynamically? Just like when you click a button the initial image will update, so on. This is working pen describing my problem, kindly check https://codepen.io/smplyjr/pen/LmENoM?editors=1010

Thanks 👍

**Comment by smplyjr** [comment]
Yup, I think thats the issue. My mistake not making look over it on the docs. Thanks mate! Cheers. :tada:

**Comment by zhanziyang** [comment]
@smplyjr It's not normal. When do you call generateBlob? Can you provide demo that reproduce?

**Comment by zhanziyang** [comment]
On vue-croppa 2.x: Pass a new URL, data URL, or  to the  prop, or call  programmatically.\n\nClosing as answered. Please feel free to open a new issue if this doesn't cover your use case.

**Comment by zhanziyang** [comment]
On vue-croppa 2.x: Pass a new URL, data URL, or `HTMLImageElement` to the `initialImage` prop, or call `setFile(file)` programmatically.

Closing as answered. Please feel free to open a new issue if this doesn't cover your use case.
